apps/wallet/src/ui/app/components/receipt-card/StakeTxnCard.tsx

```diff
@@ -30,7 +30,11 @@ export function StakeTxnCard({ event }: StakeTxnCardProps) {
         system?.activeValidators?.length || null
     );
 
-    const apy = rollingAverageApys?.[validatorAddress] || 0;
+    const apy =
+        rollingAverageApys?.[validatorAddress] ||
+        rollingAverageApys?.[validatorAddress] === 0
```

Comment:
Just do `?? null`, which will handle the `0` case.

Reply:
Just saw this, I will update the pr